Updated Hardware Deployment (#1378)
btcpayserver/btcpayserver-doc: docs/Deployment/LightningInABox.md

# Hardware Deployment

[Lightning in a Box](https://lightninginabox.co)
"Your hardware. Your node. Your keys. Your Bitcoin".

In order for your Bitcoin experience to be truly self-sovereign and trustless you should consider **running nodes on your own hardware** and internet connection.
**BTCPay Server is an excellent way to run both Bitcoin & Lightning nodes**.
Not only are you validating transactions you also get the ability to accept base layer Bitcoin and second layer Lightning payments.

> Followng are instructions to install and host your very own BTCPay Server.

Reviewer: ```suggestion
Following are instructions to install and host your very own BTCPay Server.
```

The process is basically the following:

1. Purchase and assemble hardware.
2. Install base Operating System and configure networking.
3. Install BTCPayServer-Docker.

> **BTCPayServer can be installed on the following hardware**. The end result is a small, quiet node that is fast enough to sync from block zero. The total cost is approximately $300. 

Reviewer: ```suggestion
**BTCPay Server can be installed on the following hardware**. The end result is a small, quiet node that is fast enough to sync from block zero. The total cost is approximately $300. 
```

1. [BeeLink S12 - Mini PC - $169.00](https://www.amazon.com/dp/B0C89TQ1YF?ref=nb_sb_ss_w_as-reorder-t1_k0_1_4&amp=&crid=SHKYOXZIRAO0&amp=&sprefix=beel)
3. [WD Blue 2TB SSD - $129.00](https://www.amazon.com/Western-Digital-SA510-Internal-Solid/dp/B0C14TF467/ref=sr_1_3?crid=2WDY52E7ESSEB&dib=eyJ2IjoiMSJ9.MBxkb5ZIvwjKXOzscB0GUvsbhX1rVhilXNFzID6n0xHORsDBPkIxQhIixVuiLY9I16rlFs5COExAAD8761Do-tzuAnZiutbqN-KM9rAL4zCw94kA_ArCJeR_RTDynZbiXf2Phnahw1Gw2dqXVek3p0dpe6_a_fbJrqx4BRaieoYo0zj1mX6YPGaYZAmF2Vf_Quk1TrkARk6s1_wZ0vFUw7EWdjKJ9hmNLxPWMfADML90A1rXk8gSCcRnwV2jdzN7jCfg2_urfJZ3IWOW5X3iwnP7s-vSec88PGmQ3RhS-Rc.sEURveFhiTAHYwZQdwyJX72hpWL5UgD_3tEPet747oE&dib_tag=se&keywords=2tb+ssd+wd+blue&qid=1710685725&s=electronics&sprefix=2tb+ssd+wd+blue%2Celectronics%2C90&sr=1-3)

Other requirements are as follows:

1. High speed internet connection.
2. Static IP
3. Domain Name

> 4. Ability to open ports (80, 443, 9735) on your router (optional, BTCPayServer can be accessed over TOR or by Dynamic DNS). 

Reviewer: ```suggestion
4. Ability to open ports (80, 443, 9735) on your router (optional, BTCPay Server can be accessed via Tor or by Dynamic DNS). 
```

The Ubuntu installation process is pretty simple and easy to follow. Here's a tutorial from the Ubuntu website. [Install Ubuntu Server](https://ubuntu.com/tutorials/install-ubuntu-server#1-overview). The BeeLink S12 ships with Windows Pre-Installed so you will have to delete the NVME partitions and install Ubuntu on that drive. 

*During the install process make sure you set the hostname to "btcpay" and enable SSH. 

### Give your LIAB a static IP address on your local network. 
There are a few different ways to do this and you will find a ton of articles online. Here's a pretty simple one to follow [How to configure a static IP address on Ubuntu 22.04](https://www.linuxtechi.com/static-ip-address-on-ubuntu-server/). To avoid conflicts with other devices on your network you should also set a "reservation" for your LIAB.

### Log into your router and forward ports 80, 443 and 9735 to your LIAB's local IP address. (optional, if using .local, or Tor only)
Every router is different and you should be able to find instructions for your router by searching for "Port Forward + your router make and model".

> ### Install Fail2ban and GIT. OpenSSH server allows you to connect to your server using SSH clients (ie. [Putty](https://www.putty.org/)) 

Reviewer: ```suggestion
### Install Fail2ban and Git

OpenSSH server allows you to connect to your server using SSH clients (ie. [Putty](https://www.putty.org/)) 
```

> Fail2ban bans IP's that attempt to connect to your server and show malicious signs. GIT allows you to clone and manage repositories on github.com.

Reviewer: ```suggestion
Fail2ban bans IP's that attempt to connect to your server and show malicious signs.
Git allows you to clone and manage repositories on github.com.
```
